docs/Using-Fleet/MDM-setup.md

```diff
@@ -7,41 +7,89 @@ In Fleet, MDM features are supported for Macs running macOS 12 (Monterey) and hi
 
 # Overview
 
-To use Fleet's MDM features you first first have to [deploy Fleet](../Deploying/Introduction.md) and [add your hosts to Fleet](./Adding-hosts.md).
-
 MDM features require Apple's Push Notification service (APNs) to control and secure Apple devices. This guide will walk you through how to generate and upload a valid APNs certificate to Fleet in order to use Fleet's MDM features.
 
 [Automated Device Enrollment](https://support.apple.com/en-us/HT204142) allows Macs to automatically enroll to Fleet when they are first set up. This guide will walk you through how to connect Apple Business Manager (ABM) to Fleet. Note that this is only required if you are using Automated Device Enrollment AKA Device Enrollment Program (DEP) AKA "Zero-touch."
 
-> Only users with the admin role in Fleet can complete these setups.
+# Requirements
+To use Fleet's MDM features you must have:
+1. A [deployed Fleet instance](../Deploying/Introduction.md)
+2. A Fleet user with the admin role
 
 ## Apple Push Notification service (APNs)
+Apple uses APNs to authenticate and manage interactions between Fleet and the host. 
+
+To connect Fleet to APNs, we will do the following steps:
+1. Generate four required files
+2. Generate an APNs certificate from APNs
+3. Configure Fleet with the required files
 
-To connect Fleet to Apple, get these four files using the Fleet UI or the `fleetctl` command-line interface: An APNs certificate, APNs private key, Simple Certificate Enrollment Protocol (SCEP) certificate, and SCEP private key.
+### Step 1: generate required files
+For the MDM protocol to function, we need to generate the four following files:
+1. APNs certificate 
+2. APNs private key 
+3. Simple Certificate Enrollment Protocol (SCEP) certificate 
+4. SCEP private key
+
+The APNs certificates serves as authentication between Fleet and Apple, while the SCEP certificates serve as authentication between Fleet and hosts.
 
 To do this, choose the "Fleet UI" or "fleetctl" method and follow the steps below.
 
 Fleet UI:
 
-1. Head to the **Settings > Integrations > Mobile device management (MDM)** page. Users with the admin role can access the settings pages.
-
-2. Follow the instructions under **Apple Push Certificates Portal**.
+1. Head to the **Settings > Integrations > Mobile device management (MDM)** page.
+2. Under **Apple Push Certificates Portal**, select "Request," then fill out the form. This should generate three files and send an email to you with an attached CSR file.
```

Comment:
```suggestion
2. Under **Apple Push Certificates Portal**, select **Request**, then fill out the form. This should generate three files and send an email to you with an attached CSR file.
```

Heads up, I think we're using **bold** styles to call out Fleet UI elements elsewhere in the docs.